testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -500,3 +515,15 @@ def test_discrete_Frechet_concentric_circles(self, path_1, path_2):
         expected = 4.5
         actual = PSA.discrete_frechet(path_1, path_2)
         assert_almost_equal(actual, expected)
+
+
+def test_get_num_atoms_path_no_path_data():
+    """Test that ValueError is raised when no path data i.e. user did not
+    run Path.to_path()"""
+    match_exp = "No path data"
+    with pytest.raises(ValueError, match=match_exp):
+        universe1 = mda.Universe(PSF, DCD)
```

Comment:
If you are going to use these `Universes` more than once probably best to make them a fixture

Reply:
Done! Thanks for the feedback! I also reorganized the tests under class TestPSAExceptions and added 2 new tests for psa.get_num_paths and psa.get_paths as I realized those functions needed similar testing.